GitOut/Features/Git/Stage/GitStagePage.xaml

```diff
@@ -571,13 +571,37 @@
                                 <ColumnDefinition Width="Auto"/>
                                 <ColumnDefinition Width="*"/>
                                 <ColumnDefinition Width="Auto"/>
+                                <ColumnDefinition Width="Auto"/>
                             </Grid.ColumnDefinitions>
                             <StackPanel Orientation="Horizontal">
                                 <CheckBox Style="{StaticResource MaterialCheckBoxStyle}" Content="Amend" IsChecked="{Binding AmendLastCommit}"/>
                                 <CheckBox Style="{StaticResource MaterialCheckBoxStyle}" Content="Checkout new branch" IsChecked="{Binding CheckoutBranchBeforeCommit}"/>
                             </StackPanel>
-                            <Button Grid.Column="2" Style="{StaticResource MaterialIconButtonStyle}" Command="{Binding CommitCommand}" ToolTip="Commit">
-                                <Path Fill="{StaticResource MaterialForegroundBase}" Data="{StaticResource ArrowRightBox}"/>
+                            <Button Grid.Column="2" Command="{Binding StashIndexCommand}" ToolTip="Stash index">
+                                <Button.Style>
+                                    <Style BasedOn="{StaticResource MaterialIconButtonStyle}" TargetType="{x:Type Button}">
```

Comment:
Since this is the same as the other "specific style", how about we create a shared style in this files resource group?

Reply:
much better :star: